docs/data-model.html

```diff
@@ -0,0 +1,868 @@
+"description": "My ropsten testnet account.",
+"privateKeyBrowser": "urn:metamask:0x774477c4cd54718d32d4df393415796b9bfcb63c"
+}
+</pre
+      >
+    </section>
+
+    <section>
+      <h3 id="Mnemonic">Mnemonic</h3>
+      <p>An ordered list of memorable words.</p>
+
+      <p>MAY be used as a source of memorable <a>Entropy</a>.</p>
+      <p>
+        MAY be used as input to a
+        <a href="https://en.wikipedia.org/wiki/Key_derivation_function">KDF</a
+        >.
+      </p>
+
+      <p>Here is an example of storing a mnemonic as a <a>Secret</a>.</p>
+
+      <pre class="example highlight" title="A mnemonic example">
+{
+"@context": ["https://w3id.org/wallet/v1"],
+"id": "urn:uuid:c410e44a-9525-11ea-bb37-0242ac130002",
+"name": "My Ropsten Mnemonic 1",
+"image": "https://via.placeholder.com/150",
+"description" : "For testing only, totally compromised.",
+"tags": ["professional", "organization", "compromised"],
+"correlation": ["4058a72a-9523-11ea-bb37-0242ac130002"],
+"type": "Mnemonic",
+"value": "humble piece toy mimic miss hurdle smile awkward patch drama hurry mixture"
+}
+</pre
+      >
+    </section>
+
+    <section>
+      <h3 id="Entropy">Entropy</h3>
+
+      <p>
+        MAY be used as input to a
+        <a href="https://en.wikipedia.org/wiki/Key_derivation_function">KDF</a
+        >.
+      </p>
+
+      <p>Here is an example of seed entropy as a <a>Secret</a>.</p>
+
+<pre class="example highlight" title="A hex encoded seed stored in google cloud">
+{
+"@context": [ "https://w3id.org/wallet/v1" ],
+"id": "urn:google:projects/555555555555/secrets/seed/versions/1",
+"type": "GoogleCloudSecret",
+"tags": [ "google" ],
+"value": "7052adea8f9823817065456ecad5bf24dcd31a698f7bc9a0b5fc170849af4226"
+}
+</pre
+                >
+
+      <p>Here is an example of storing entropy as a <a>Secret</a>.</p>
+
+      <pre class="example highlight" title="An entropy example">
+{
+"@context": ["https://w3id.org/wallet/v1"],
+"id": "urn:uuid:c410e44a-9525-11ea-bb37-0242ac130002",
+"name": "My Test Entropy 1",
+"image": "https://via.placeholder.com/150",
+"description" : "For testing only, totally compromised.",
+"tags": ["professional", "organization", "compromised"],
+"correlation": ["4058a72a-9523-11ea-bb37-0242ac130002"],
+"type": "Entropy",
+"multibase": "z6MkuVZVSKLUNHRjXBXcEwvG5yiNHLhzBeXRjAkwmgSYensE"
+}
+</pre
+      >
+    </section>
+    
+
+    <section>
+      <h3>Connection</h3>
+
+      <p>Information about a relationship between identifiers.</p>
+
+      <div class="issue" data-number="4">
+        Need to capture how connections are represented today.
+      </div>
+
+      <pre class="example highlight" title="A connection example">
+{
+"@context": ["https://w3id.org/wallet/v1"],
+"id": "urn:uuid:c410e44a-9525-11ea-bb37-0242ac130006",
+"name": "My Health Record Certifier",
+"image": "https://via.placeholder.com/150",
+"description" : "The identifier that issues health record credentials.",
```

Comment:
The identifier doesn't issue anything. This might be the identifier **of the entity** that issues...

Reply:
agreed!